ONSdigital/design-system
# A11y fix for: Information Conveyed Visually (A): Information panels are conveyed via visual styling alone and are not programmatically determinable

### What is the context of this PR?
DAC failed on WCAG 2.1 guideline 1.3.1 Info and Relationships (Level A) due to panel information being conveyed by visual styling alone and not programmatically determinable.

This PR follows on from #1320 by fixing an issue, and also adding prefixes for the remaining panels which have visual only cues to indicate type of information contained (e.g. error, warning etc.)

In turn, also fixes DAC issue raised [Validation Pattern (Usability) The purpose of error messages was not clearly presented to users](https://trello.com/c/LbUmPM30/920-validation-pattern-usability-the-purpose-of-error-messages-was-not-clearly-presented-to-users) 

## Changes
- Fixed invalid html in warning panel (`<div>` inside `<strong>`)
- Added default and overridable hidden prefixes for default, branded, error, success panels

### How to review 
- Check all panels with screen readers
- Check macro template logic for jinja2 compatibility issues
- Check valid html
